### PR TITLE
Docker image versions updated for tests vm module with terraform-0.12 

### PR DESCRIPTION
**Pull request by ingwarr**
Fixes tests for https://github.com/terraform-google-modules/terraform-google-vm/pull/16

### CODE REVIEW

File: infra/concourse/pipelines/terraform-google-vm.yml [hunk 1]
```diff
@@ -17,15 +17,15 @@ resources:
     type: docker-image
     source:
       repository: gcr.io/cloud-foundation-cicd/cft/lint
-      tag: 1.0.0
+      tag: 2.5.0
```
Comment (aaron-lane) [inline infra/concourse/pipelines/terraform-google-vm.yml]: This should be 2.4.0; 2.5.0 is not ready for use yet.

File: infra/concourse/pipelines/terraform-google-vm.yml [hunk 2]
```diff
@@ -17,15 +17,15 @@ resources:
...
       username: _json_key
       password: ((sa.google))
 
   - name: integration-test-image
     type: docker-image
     source:
       repository: gcr.io/cloud-foundation-cicd/cft/kitchen-terraform
-      tag: 1.2.0
+      tag: 2.4.0
```
Comment (aaron-lane) [inline infra/concourse/pipelines/terraform-google-vm.yml]: This should be 2.3.0; 2.4.0 is not ready for use yet.

Comment (ingwarr) [reply infra/concourse/pipelines/terraform-google-vm.yml]: Done

File: infra/concourse/pipelines/terraform-google-vm.yml (hunk 1 above)
Comment (ingwarr) [reply infra/concourse/pipelines/terraform-google-vm.yml]: Done

**Pull request merged**